src/Compilers/VisualBasic/Test/Syntax/Parser/ParseErrorTests.vb

```diff
@@ -1739,6 +1739,18 @@ BC30306: Array subscript expression missing.
                              </errors>)
     End Sub
 
+    <Fact()>
+    Public Sub BC30241ERR_ExpectedNamedArgument_VB15_6()
+        Dim code = <![CDATA[
+<Attr1(1, b:=2, 3, e:="Scen1")>
+Class Class1
+
+End Class
+            ]]>.Value
+
+        ParseAndVerify(code, LanguageVersion.VisualBasic15_6, False, Nothing) ' No parsing errors
```

Comment:
Yes, it seems that the named argument syntax in attributes in VB refers to fields/properties, not constructor parameters. 

```VB
Imports System

'error BC30661: Field or property 'a' is not found.
'error BC30455: Argument not specified for parameter 'a' of 'Public Sub New(a As Integer)'.
<Attr1(a:=1)>
Class Class1
End Class

Class Attr1
    Inherits Attribute

    Public Sub New(a As Integer)    
    End Sub
End Class

```

---
In reply to: [122319535](https://github.com/dotnet/roslyn/pull/20182#discussion_r122319535) [](ancestors = 122319535)